test/extensions/filters/http/aws_lambda/BUILD

```diff
@@ -23,6 +23,20 @@ envoy_extension_cc_test(
     ],
 )
 
+envoy_extension_cc_test(
+    name = "aws_lambda_filter_integration_test",
+    srcs = ["aws_lambda_filter_integration_test.cc"],
+    coverage = False,
```

Comment:
Put a TODO comment here discussing why this is disabled.

Reply:
Done.